### PR TITLE
Update l10n.js - Korean

### PR DESCRIPTION
**Pull request by pensivej**
I am working on translating w3c using respec, and TOC is not rendered due to lack of I10n when I change to lang=ko.
So I have added translations into korean

### CODE REVIEW

**Comment by saschanaz** [comment]
Hello @pensivej, thank you for working on this!

Do you have a public translation that I can check out now?

File: src/core/l10n.js [hunk 1]
```diff
@@ -44,6 +44,11 @@ const base = {
 
 const ko = {
   abstract: "요약",
+  participate: "참여",
+  references: "참조",
+  toc: "목차",
+  example: "예시",
+  informative: "정보",
```
Comment (saschanaz) [inline src/core/l10n.js]: These two should go `src/core/example.js` and `src/core/informative.js`.

**Comment by pensivej** [comment]
Here it is:
https://ssimeetupkorea.github.io/did-core/

For now, it is set to html=en

File: src/core/l10n.js (hunk 1 above)
Comment (sidvishnoi) [reply src/core/l10n.js]: I should probably finish up https://github.com/w3c/respec/issues/2084 first

Comment (pensivej) [reply src/core/l10n.js]: resolved

**Pull request merged**